<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0" idbasedtr="false">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>480</width>
    <height>456</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Rigidity Experiment Trajectory Generator</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="1" column="0">
     <widget class="QGroupBox" name="groupBox_5">
      <property name="title">
       <string>Menu</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QPushButton" name="preview_button">
         <property name="text">
          <string>Preview</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="generate_button">
         <property name="text">
          <string>Generate</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="save_button">
         <property name="text">
          <string>Save</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="load_button">
         <property name="text">
          <string>Load</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QGroupBox" name="groupBox_2">
      <property name="title">
       <string>Ramp</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="7" column="0">
        <widget class="QLabel" name="label_12">
         <property name="text">
          <string>Plateau Time</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_9">
         <property name="text">
          <string>Level</string>
         </property>
        </widget>
       </item>
       <item row="12" column="0" colspan="4">
        <widget class="QLabel" name="label_7">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="6" column="0">
        <widget class="QLabel" name="label_10">
         <property name="text">
          <string>Pause Time</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="label_8">
         <property name="text">
          <string>Slope</string>
         </property>
        </widget>
       </item>
       <item row="0" column="3">
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="7" column="1" colspan="2">
        <widget class="QDoubleSpinBox" name="Ramp_Plateau_Time">
         <property name="minimum">
          <double>0.250000000000000</double>
         </property>
         <property name="maximum">
          <double>10.000000000000000</double>
         </property>
         <property name="singleStep">
          <double>0.050000000000000</double>
         </property>
         <property name="value">
          <double>2.500000000000000</double>
         </property>
        </widget>
       </item>
       <item row="6" column="1">
        <widget class="QDoubleSpinBox" name="Ramp_Min_Pause_Time">
         <property name="singleStep">
          <double>0.250000000000000</double>
         </property>
         <property name="value">
          <double>2.500000000000000</double>
         </property>
        </widget>
       </item>
       <item row="6" column="2">
        <widget class="QDoubleSpinBox" name="Ramp_Max_Pause_Time">
         <property name="singleStep">
          <double>0.250000000000000</double>
         </property>
         <property name="value">
          <double>5.000000000000000</double>
         </property>
        </widget>
       </item>
       <item row="10" column="1" colspan="2">
        <widget class="QSpinBox" name="Ramp_Repetition">
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="value">
          <number>5</number>
         </property>
        </widget>
       </item>
       <item row="10" column="0">
        <widget class="QLabel" name="label_20">
         <property name="text">
          <string>Repetition</string>
         </property>
        </widget>
       </item>
       <item row="11" column="0" colspan="2">
        <widget class="QRadioButton" name="Ramp_Order">
         <property name="text">
          <string>Random Order</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="0" column="1" colspan="2">
        <widget class="QLineEdit" name="Ramp_Slope">
         <property name="text">
          <string>[0.1, 0.25, 0.5]</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1" colspan="2">
        <widget class="QLineEdit" name="Ramp_Level">
         <property name="text">
          <string>[0.5, 1]</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="QGroupBox" name="groupBox_3">
      <property name="title">
       <string>Chirp</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_4">
       <item row="9" column="2" colspan="2">
        <widget class="QDoubleSpinBox" name="Chirp_Max_Amplitude">
         <property name="maximum">
          <double>1.000000000000000</double>
         </property>
         <property name="singleStep">
          <double>0.050000000000000</double>
         </property>
         <property name="value">
          <double>1.000000000000000</double>
         </property>
        </widget>
       </item>
       <item row="11" column="0">
        <widget class="QLabel" name="label_11">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="5" column="2" colspan="2">
        <widget class="QDoubleSpinBox" name="Chirp_Start_Time">
         <property name="minimum">
          <double>2.500000000000000</double>
         </property>
         <property name="maximum">
          <double>10.000000000000000</double>
         </property>
         <property name="singleStep">
          <double>0.050000000000000</double>
         </property>
         <property name="value">
          <double>5.000000000000000</double>
         </property>
        </widget>
       </item>
       <item row="7" column="0">
        <widget class="QLabel" name="label_18">
         <property name="text">
          <string>Plateau Time</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_13">
         <property name="text">
          <string>Max Frequency</string>
         </property>
        </widget>
       </item>
       <item row="5" column="0">
        <widget class="QLabel" name="label_14">
         <property name="text">
          <string>Start Time</string>
         </property>
        </widget>
       </item>
       <item row="6" column="0">
        <widget class="QLabel" name="label_15">
         <property name="text">
          <string>Ramp Time</string>
         </property>
        </widget>
       </item>
       <item row="9" column="0">
        <widget class="QLabel" name="label_17">
         <property name="text">
          <string>Max Amplitude</string>
         </property>
        </widget>
       </item>
       <item row="10" column="0">
        <widget class="QLabel" name="label_21">
         <property name="text">
          <string>Repetition</string>
         </property>
        </widget>
       </item>
       <item row="10" column="2" colspan="2">
        <widget class="QSpinBox" name="Chirp_Repetition">
         <property name="value">
          <number>5</number>
         </property>
        </widget>
       </item>
       <item row="6" column="2" colspan="2">
        <widget class="QDoubleSpinBox" name="Chirp_Ramp_Time">
         <property name="minimum">
          <double>10.000000000000000</double>
         </property>
         <property name="maximum">
          <double>360.000000000000000</double>
         </property>
         <property name="singleStep">
          <double>10.000000000000000</double>
         </property>
         <property name="value">
          <double>60.000000000000000</double>
         </property>
        </widget>
       </item>
       <item row="7" column="2" colspan="2">
        <widget class="QDoubleSpinBox" name="Chirp_Plateau_Time">
         <property name="minimum">
          <double>2.500000000000000</double>
         </property>
         <property name="maximum">
          <double>10.000000000000000</double>
         </property>
         <property name="singleStep">
          <double>0.500000000000000</double>
         </property>
         <property name="value">
          <double>5.000000000000000</double>
         </property>
        </widget>
       </item>
       <item row="1" column="2" colspan="2">
        <widget class="QDoubleSpinBox" name="Max_Frequency">
         <property name="minimum">
          <double>0.000000000000000</double>
         </property>
         <property name="singleStep">
          <double>0.050000000000000</double>
         </property>
         <property name="value">
          <double>1.000000000000000</double>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="1" column="2">
     <widget class="QGroupBox" name="groupBox_4">
      <property name="title">
       <string>Output </string>
      </property>
      <layout class="QGridLayout" name="gridLayout_6">
       <item row="2" column="0">
        <widget class="QLabel" name="label_19">
         <property name="text">
          <string>CHIRP BLOCK TIME</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QLCDNumber" name="Chirp_Display">
         <property name="digitCount">
          <number>8</number>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLCDNumber" name="Ramp_Display">
         <property name="digitCount">
          <number>8</number>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_16">
         <property name="text">
          <string>RAMP BLOCK TIME</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
